appveyor.yml

```diff
@@ -23,13 +22,11 @@ build_script:
   - qbs setup-toolchains --type mingw %MINGW%\bin\g++.exe mingw
   - qbs setup-qt %QTDIR%\bin\qmake.exe qt
   - qbs config defaultProfile qt
-  - qbs install --install-root install-root release
+  - qbs install --all-products --install-root install-root release
 
 after_build:
-  - pushd %APPVEYOR_BUILD_FOLDER%\dist\win
-  - set TILED_BUILD_DIR=%APPVEYOR_BUILD_FOLDER%\install-root
-  - set ARCH=32
-  - '%NSIS_ROOT%\makensis tiled.nsi'
+  - cd qt-release
+  - cd dist*
```

Comment:
I know this is weird but cmd is incapable of understanding something like `*\dist*\tiled-*-setup.exe`